docs/data-type-support.rst

```diff
@@ -0,0 +1,6 @@
+Data type support
+******************************************
+
+hipRAND's data type support is matching cuRAND's data type support. On AMD hardware the backend is provided by rocRAND. The detailed description, which includes a comparison to cuRAND's data type support, can be found at :doc:`rocRAND's data type support page.<rocrand:data-type-support>`
```

Comment:
Recommended text,

hipRAND's data type support matches cuRAND's data type support. On AMD hardware, the backend is provided by rocRAND.

Reply:
Thanks, fixed it